lang/en/texts/data.html

```diff
@@ -129,7 +129,7 @@ <h3>If your users expect a result immediately (e.g. Nutrition apps)</h3>
 <ul>
 <li>Send nutrition facts + category &gt; get Nutri-Score</li>
 <li>Send ingredients &gt; get the NOVA group (about food ultra-processing), additives, allergens, normalized ingredients, vegan, vegetarian&hellip;</li>
-<li>Send category + labels &gt; soon get the Eco-Score (about environmental impact)</li>
+<li>Submit category + labels &gt; soon get the Eco-Score (about environmental impact)</li>
```

Comment:
I'm kind of wondering if we should spell out `+` and `&` instead. What do you think?

Reply:
Agree